spotty/spotty/UIComponents/Styling/Enums/ImageStyle.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ImageStyle.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 20.06.22.
+//
+
+import UIKit
+
+enum ImageStyle {
+    
+    case mediumCellImage
+    case largeCellImage
+    
+    var clipsToBounds: Bool {
+        switch self {
```

Comment:
When you have only one value you can directly return in here and in the property below. We will introduce the switch statement when we have second value.

Reply:
done